include/grpcpp/impl/codegen/server_context.h

```diff
@@ -226,6 +226,8 @@ class ServerContext {
   /// Async only. Has to be called before the rpc starts.
   /// Returns the tag in completion queue when the rpc finishes.
   /// IsCancelled() can then be called to check whether the rpc was cancelled.
+  /// TODO(vpai): The tag won't be returned if the call never starts.
```

Comment:
I go by vjpai ...

Also, can you make it clear that this is a bug? Something like 

```
TODO(vjpai): Fix this so that the tag is returned even if the call never starts
```

Reply:
Done. 

Sorry, I thought you are using the internal ID.